samples/csharp_dotnetcore/17.multilingual-bot/appsettings.json

```diff
@@ -0,0 +1,5 @@
+﻿{
+  "botFilePath": "BotConfiguration.bot",
+  "botFileSecret": "",
+  "translatorKey": "<Your translation key here>"
```

Comment:
You could consider adding the translator key as a generic service using MSBot as well.